green_city/tests/api/get_eco_news_likes_test.py

```
@@ -0,0 +1,88 @@
+import pytest
+import requests
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)